generators/uimodule/ui5Libs.js

```diff
@@ -50,19 +52,26 @@ export default class extends Generator {
 				manifestJSON["sap.ui5"]["dependencies"]["minUI5Version"] = dependencies["SAPUI5"]
 				break
 			case "Local resources (OpenUI5)":
+				localResources = true
 				ui5Yaml.framework.name = "OpenUI5"
 				ui5Yaml.framework.version = dependencies["OpenUI5"]
-				deleteProxyToUI5(ui5Yaml)
 				manifestJSON["sap.ui5"]["dependencies"]["minUI5Version"] = dependencies["OpenUI5"]
 				break
 			case "Local resources (SAPUI5)":
+				localResources = true
 				ui5Yaml.framework.name = "SAPUI5"
 				ui5Yaml.framework.version = dependencies["SAPUI5"]
-				deleteProxyToUI5(ui5Yaml)
 				manifestJSON["sap.ui5"]["dependencies"]["minUI5Version"] = dependencies["SAPUI5"]
 				break
 		}
 
+		if (localResources && this.options.config.platform !== "SAP Build Work Zone, standard edition") {
+			// only if no flpSandbox.html exists (via preview-middleware), as it requires proxy to fetch all libs
```

Comment:
@nicoschoenteich I'm not sure if I understood this comment 🤔 The preview-middleware can be used with or without proxy. In case of usage without proxy (UI5 sources served via tooling/npmjs) you just need to ensure that the libs needed for the sandbox FLP are part of the `framework.libraries` list of `ui5*.yaml`

afaik these are
```
    - name: sap.ushell                    # FLP sandbox
    - name: sap.suite.ui.generic.template # dependency of ushell
    - name: sap.ui.generic.template       # dependency of sap.suite.ui.generic.template
    - name: sap.tnt                       # specialized controls
```

Reply:
Yep, I know we can also serve the libs locally, but I would like to keep the list of local libs as crisp as possible. Also, I would like to not provide a launchpad if the platform is not `SAP Build Work Zone, standard edition`, so that's why we do the `deleteProxyToUI5()`.